gips/data/landsat/acolite.cfg

```diff
@@ -0,0 +1,132 @@
+;; ACOLITE Settings
+version=20160905.0
+
+;; Inputfile (imgdir=)
+;; Landsat: input file directory or MTL file
+;; Sentinel: .SAFE file or granule
+;; imgdir=
+
+;; limit = ROI bounding box lat1,lon1,lat2,lon2 (decimal degrees)
+limit=
```

Comment:
Huh?  This file?  It is a configuration that is fed to ACOLITE, that needs to be tweaked on the fly for each job.